app/dashboard/tasks.py

```diff
@@ -15,6 +15,34 @@
 LOCK_TIMEOUT = 60 * 2
 
 
+@app.shared_task(bind=True, max_retries=3)
+def bounty_on_create(self, team_id, new_bounty, retry: bool = True) -> None:
+    # what has to happen that we want to chain data from one another together
```

Comment:
What do these comments mean?

Reply:
@androolloyd is this creating a channel on bounty creation ?